### ISSUE TITLE
Related Documents links - Insufficient contrast ratio

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/aria-practices#2384]
- **URL**: https://www.w3.org/WAI/ARIA/apg/example-index/landmarks/index.html
- **Accessibility failure**: contrast ratio between related documents links text color and background is not sufficient (#337ab7 foreground color and #f4f4f4 background color, with a contrast ratio of 4.14:1)  - 1.4.3: Contrast (Minimum)
- **Expected result**: contrast ratio for standard text is at least 4.5:1
- **WCAG reference**: The visual presentation of [text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-text) and [images of text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-image-of-text) has a [contrast ratio](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-contrast-ratio) of at least 4.5:1, ...
- **Issue snapshot**: <img width="1260" alt="screenshot highlighting poor contrast ratio between related documents links and background" src="https://user-images.githubusercontent.com/106394951/174055443-a6a10076-de21-4bab-a463-a1c82eb762f1.png">

Giacomo Petri
